apps/camera/js/controllers/viewfinder.js

```diff
@@ -237,10 +237,12 @@ ViewfinderController.prototype.onFacesDetected = function(faces) {
 };
 
 ViewfinderController.prototype.calculateFaceCircle = function(face) {
+  var diameter = Math.max(face.width, face.height);
+  var radius = diameter/2;
```

Comment:
So this was supposed to be &lt; space &gt; / &lt; space &gt; 2 ... but github thinks it's smarter than I am.